docs/quickstart.md

```diff
@@ -55,94 +55,81 @@ sudo cp target/x86_64-unknown-linux-musl/release/{firecracker,jailer} /usr/local
 
 cd ~
 
-# Check out containerd and build it from the v1.2.4 tag
-mkdir -p ~/go/src/github.com/containerd/containerd
-git clone https://github.com/containerd/containerd.git ~/go/src/github.com/containerd/containerd
-cd ~/go/src/github.com/containerd/containerd
-git checkout v1.2.4
-sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libseccomp-dev btrfs-progs
-make
-sudo cp bin/* /usr/local/bin
-
-cd ~
-
-# Check out runc and build it from the 6635b4f0c6af3810594d2770f662f34ddc15b40d
-# commit.  Note that this is the version described in
-# https://github.com/containerd/containerd/blob/v1.2.4/RUNC.md and
-# https://github.com/containerd/containerd/blob/v1.2.4/vendor.conf#L23
-mkdir -p ~/go/src/github.com/opencontainers/runc
-git clone https://github.com/opencontainers/runc ~/go/src/github.com/opencontainers/runc
-cd ~/go/src/github.com/opencontainers/runc
-git checkout 6635b4f0c6af3810594d2770f662f34ddc15b40d
-make static BUILDTAGS='seccomp'
-sudo make BINDIR='/usr/local/bin' install
+# Install Docker CE
+# Docker CE includes containerd, but we need a separate containerd binary, built
+# in a later step
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+apt-key fingerprint 0EBFCD88 | grep Docker || echo '**Cannot find Docker key**'
+echo "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" | \
+     sudo tee /etc/apt/sources.list.d/docker.list
+sudo DEBIAN_FRONTEND=noninteractive apt-get update
+sudo DEBIAN_FRONTEND=noninteractive apt-get \
+     install --yes \
+     docker-ce
```

Comment:
Docker's packages for Debian still recommend `aufs-tools`, which will pull in `dkms` and the whole suite of packages needed to build kernel modules. Let's avoid that by adding `aufs-tools-` to the list of packages.